scripts/ps-modules/Build/Build.psm1

```diff
@@ -267,35 +312,24 @@ function Run-InstallPackageStep
 
 function Run-PrestageAndFinalizeBuildArtifactsStep {
    param(
-      [string]$linkType,
-      [string]$buildType,
-      [string]$customTriplet,
+      [string[]]$triplets
```

Comment:
Looks like we are missing a comma.  This is producing an error on the build server: https://dev.azure.com/techsmith/ThirdParty-Packages-vcpkg/_build/results?buildId=419090&view=logs&j=c7dcf16a-b484-548d-e1f9-8e1c62e0ef67&t=cd0aa3ac-cd97-56f0-d73f-ebc6676e665d

Reply:
Ah, thanks for checking!